addons/binding/org.openhab.binding.homematic/README.md

```diff
@@ -8,14 +8,15 @@ This binding allows you to integrate, view, control and configure all Homematic
 All gateways which provides the Homematic BIN- or XML-RPC API:
 
 - CCU 1+2
+- [RaspberryMatic](https://github.com/jens-maus/RaspberryMatic)
 - [Homegear](https://www.homegear.eu)
 - [piVCCU](https://github.com/alexreinert/piVCCU)
 - [YAHM](https://github.com/leonsio/YAHM)
 - [Windows BidCos service](http://www.eq-3.de/downloads.html?kat=download&id=125)
 - [OCCU](https://github.com/eq-3/occu)
 
 The Homematic IP Access Point does not support this API and can't be used with this binding.
-But you can control Homematic IP devices with a CCU2 with at least firmware 2.17.15.
+But you can control Homematic IP devices with a CCU2 with at least firmware 2.17.15 or use [RaspberryMatic](https://github.com/jens-maus/RaspberryMatic) with the [HM-MOD-RPI-PCB](https://www.elv.de/homematic-funkmodul-fuer-raspberry-pi-bausatz.html) RF module.
```

Comment:
Consider removing the "But"
Also I had to read the sentence multiple times to understand that it is about Homematic IP. You should make this more clear! If you need my help with the README, I'd be able to help.